Solutions/FalconFriday/Analytic Rules/DotNetToJScript.yaml

```diff
@@ -0,0 +1,42 @@
+id: 9f921513-65f3-48a2-ae7d-326c5901c55e
+name: Detect .NET to JS
```

Comment:
descriptive names indicating what actions are malicious or suspicious e.g. loading of .net